src/components/sidebar.tsx

```diff
@@ -26,6 +26,12 @@ export default function Sidebar() {
         </NavLink>
       </div>
 
+      <div className="change-theme">
```

Comment:
The class name is not related to the icon, we can try a common one, say `icon-links` or `side-links` to be more descriptive and avoid confusion.

Reply:
Yes, that would be better. On it.